packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -223,3 +249,25 @@ export const listAccounts = async () => {
 
 	return accounts;
 };
+
+/**
+ * Look up the latest release of workerd and use its date as the compatibility_date
```

Comment:
Would be good to add a comment about the expected input format or something, like:
"Extracts the latest release date from workerd's version number (of the form 'major.yyyymmdd.patch')"
or something like that